Replicator/Checkpointer.cc

```diff
@@ -109,7 +109,7 @@ namespace litecore { namespace repl {
 
     size_t Checkpointer::pendingSequenceCount() const {
         LOCK();
-        return _checkpoint->pendingSequenceCount();
+        return _checkpoint ? _checkpoint->pendingSequenceCount() : 0;
```

Comment:
What reason would a checkpoint be null?  I would think we should read the checkpoint whenever we need it or we might not get an accurate count.

Reply:
It's a passive replicator, so it doesn't care about the checkpoint and never loads it.